content/blogposts/2024/sourcegraph-release-may-2024.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Sourcegraph May 2024 release"
+publishDate: 2024-05-09T10:00-07:00
+description: "The latest Sourcegraph release includes a new point-in-time search filter for Code Search."
+tags: [blog, release]
+slug: "release/May-2024"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+changelogItems:
+  - description: "Improved the performance of language stats insights by 50-70%, significantly reducing pain for customers with large repos. The timeout for language stats insights has also been raised from 3 to 5 minutes. Both are configurable using environment variables on the frontend container."
+    url: https://sourcegraph.com/docs/code_insights/explanations/administration_and_security_of_code_insights#language-stats-performance-configuration
+    category: Code Insights
+  - description: "Improved Zoekt CPU usage and scalability for customers with a large number of repos."
+    url:
+    category: Search
+  - description: "Greatly improved git blame performance on large files, reducing the time it takes for a git blame on a large file to load from several seconds down to less than 500 milliseconds."
+    url:
+    category: Search
+  - description: "Search Jobs now supports diff, commit, and path searches. Before, only file searches were supported."
+    url:
+    category: Source
+ 
+
+
+---
+
+<br/>
+
+### Point-in-time search filter for Code Search
+
+<br/>
+
+Sourcegraph 5.4.0 brings improvements to Code Search with a new search filter [rev:at.time()](https://sourcegraph.com/docs/code-search/queries#structural-search) that provides point-in-time searching. This allows for searching a repo’s state at a specific timestamp rather than at a specific commit, branch, or tag.
```

Comment:
WDYT about adding a short list showing ways to use this?

```suggestion
Sourcegraph 5.4.0 brings improvements to Code Search with a new search filter [rev:at.time()](https://sourcegraph.com/docs/code-search/queries#structural-search) that provides point-in-time searching. This allows for searching a repo’s state at a specific timestamp rather than at a specific commit, branch, or tag.

The new filter works with both timestamps and natural language:

- `rev:at.time(1 year ago)`
- `rev:at.time(3 days ago)`
- `rev:at.time(2022-05-05)`
```